README.md

```diff
@@ -27,22 +26,22 @@ A source is getting associated with a consumer group ID the first time the `Read
-| `tls.enabled`        | Defines whether TLS is enabled.                                                                                                                                                                              | false    | `false`                   |
-| `clientCert`         | A certificate for the Kafka client, in PEM format. If provided, the private key needs to be provided too.                                                                                                    | false    |                           |
-| `clientKey`          | A private key for the Kafka client, in PEM format. If provided, the certificate needs to be provided too.                                                                                                    | false    |                           |
-| `caCert`             | The Kafka broker's certificate, in PEM format.                                                                                                                                                               | false    |                           |
-| `insecureSkipVerify` | Controls whether a client verifies the server's certificate chain and host name. If `true`, accepts any certificate presented by the server and any host name in that certificate.                           | false    | `false`                   |
-| `saslMechanism`      | SASL mechanism to be used. Possible values: PLAIN, SCRAM-SHA-256, SCRAM-SHA-512. If empty, authentication won't be performed.                                                                                | false    |                           |
-| `saslUsername`       | SASL username. If provided, a password needs to be provided too.                                                                                                                                             | false    |                           |
-| `saslPassword`       | SASL password. If provided, a username needs to be provided too.                                                                                                                                             | false    |                           |
+| name                 | description                                                                                                                                                                                                  | required | default value                |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|------------------------------|
+| `servers`            | Servers is a list of Kafka bootstrap servers, which will be used to discover all the servers in a cluster.                                                                                                   | true     |                              |
```

Comment:
There's a few Kafka references left in here.

Reply:
+1

Reply:
done